docs/Makefile

```diff
@@ -114,7 +114,7 @@ list-todos: html
 # Generate the ScalaDoc and the JavaDoc, for the old versions
 all_javascaladocs:
 	@echo "Building old java and scala docs"
-	(yq e '.content.sources[0].branches.*' docs-source/site.yml | xargs -n1 ./build-javascaladocs.sh)
```

Comment:
The fix starts here: `.content.sources[0].branches.*` is not a valid expression. It should be `.content.sources[0].branches[]` as done here:

https://github.com/lightbend/cloudflow/pull/1088/files#diff-c9354f203689690fae42eccaf90c1b0879bb7a4efe5ba5d5666a69e0b6e0e4efR17

But I'm moving all of this to the script.